.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Python Dev Container",
+    "build": {
+        "dockerfile": "Dockerfile"
+    },
+    "customizations": {
+        "vscode": {
```

Comment:
@GalenReich Idk what IDE you use, but if it's not VSCode, you can add relevant extensions here!